feedbackapi/opgfeedbackapi/get_secret.py

```diff
@@ -0,0 +1,42 @@
+import boto3
```

Comment:
Personally, I would put this script into CI to fetch the secret while building the containers. The secret would then be put into an environment variable which is read by the containers. (Any environment variable passed to an image when it's built is "baked into" the image.)

Locally, there's no need to do any of this, as you put the shared secret into each container via docker-compose.yml as environment variables. This also removes the need for the localstack and aws-cli containers; and also removes the need for a conditional in get_client() as you only lookup the secret in non-local envs.

Reply:
As discussed with @charlesmmarshall at standup,  WebOps do now wish us to regularly poll Secrets Manager, rather than the old way of getting it on container build or startup and putting in an env var. So having this mocked locally makes it resemble what actually will happen in prod.  I've made a story for ensuring the efficiency of this : https://opgtransform.atlassian.net/browse/LPAL-953